doc/getting-started/apl-nuc.rst

```diff
@@ -364,6 +364,89 @@ For hugeTLB to work, you'll need to reserve huge pages:
+      [  565.696139] hid-generic 0003:03F0:0024.0006: input,hidraw0: USB HID v1.10 Keyboard [CHICONY HP Basic USB Keyboard] on usb-0000:00:14.0-3/input0
+      [ 1000.587071] usb 1-2: new low-speed USB device number 9 using xhci_hcd
+      [ 1000.719824] usb 1-2: New USB device found, idVendor=046d, idProduct=c315
+      [ 1000.719934] usb 1-2: New USB device strings: Mfr=1, Product=2, SerialNumber=0
+      [ 1000.720048] usb 1-2: Product: Logitech USB Keyboard
+      [ 1000.720143] usb 1-2: Manufacturer: Logitech
+      [ 1000.724286] input: Logitech Logitech USB Keyboard as /devices/pci0000:00/0000:00:14.0/usb1/1-2/1-2:1.0/0003:046D:C315.0007/input/input10
+      [ 1000.776433] hid-generic 0003:046D:C315.0007: input,hidraw2: USB HID v1.10 Keyboard [Logitech Logitech USB Keyboard] on usb-0000:00:14.0-2/input0
+      [ 1008.387071] usb 1-1: new low-speed USB device number 10 using xhci_hcd
+      [ 1008.516312] usb 1-1: New USB device found, idVendor=046d, idProduct=c077
+      [ 1008.516421] usb 1-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
+      [ 1008.516536] usb 1-1: Product: USB Optical Mouse
+      [ 1008.516610] usb 1-1: Manufacturer: Logitech
+      [ 1008.519459] input: Logitech USB Optical Mouse as /devices/pci0000:00/0000:00:14.0/usb1/1-1/1-        1:1.0/0003:046D:C077.0008/input/input11
+      [ 1008.519714] hid-generic 0003:046D:C077.0008: input,hidraw3: USB HID v1.11 Mouse [Logitech USB Optical Mouse] on usb-   0000:00:14.0-1/input0
+
+#. From ``dmesg`` info, you can easly find specific USB device info connected to certain port
+   for example::
+    
+      keyboard 1#: usb 1-3
+      mouse 1#: usb 1-4 
+      keyboard 2#: usb 1-2
+      mouse 2#: usb 1-1
+   
+   Now you can define which keyboard and mouse to be assigned for UOS, which for SOS. 
+   You'll need this device information in the next step.
+
+#. Supposing you assign keyboard 2# and mouse 2# to the UOS. Use a text editor to modify
+   ``/usr/share/acrn/samples/nuc/launch_uos.sh`` and add the line (using
+   the keyboard and mouse identified in your dmesg output)::
+
+      -s 9, xhci, 1-1:1-2 \
+
+   Save the file, exit the editor, and run the ``sync`` command to ensure
+   any pending write buffers are written to disk. 
+  
+   by default, keyboard 1# and mouse 1# connected to Device be used for SOS without any additinoal configurartion
+   keyboard 2# and mouse 2# to UOS by modifying launch_uos.sh
+    
+   .. code-block:: none
+
+      # cd /usr/share/acrn/samples/nuc/
+      # vim lauch_uos.sh
+      # add below line into the file
+      # -s 9, xhci, 1-1:1-2 \
+      # sync
+  
+   .. note::
+     you can check and assign USB keyboard and USB mouse to UOS with modifying -s 9, xhci 1-x:1-y \
+     according to your USB device info
+     
+     there is an known issue to use USB keyboard and mouse in UOS, you have to unplug and plug back 
```

Comment:
If you're not going to reference a GitHub issue, I'd recommend changing this note to:

     You may have to unplug and plug in the keyboard and mouse
     assigned to the UOS after launching the UOS. 

Is the order of plugging these devices back in going to affect the changes made to launch_uos.sh?

Reply:
no any impact for the device info plug back into same port,